src/__tests__/no-act.js

```diff
@@ -7,3 +13,69 @@ test('act works even when there is no act from test utils', () => {
   act(callback)
   expect(callback).toHaveBeenCalledTimes(1)
 })
+
+test('async act works when it does not exist (older versions of react)', async () => {
+  jest.spyOn(console, 'error').mockImplementation(() => {})
+  const callback = jest.fn()
+  await asyncAct(async () => {
+    await Promise.resolve()
+    await callback()
+  })
+  expect(console.error).toHaveBeenCalledTimes(0)
+  expect(callback).toHaveBeenCalledTimes(1)
+
+  callback.mockClear()
+  console.error.mockClear()
+
+  await asyncAct(async () => {
+    await Promise.resolve()
+    await callback()
+  })
+  expect(console.error).toHaveBeenCalledTimes(0)
+  expect(callback).toHaveBeenCalledTimes(1)
+
+  console.error.mockRestore()
```

Comment:
minor, but, technically shouldn't this be in a before or after hook so it doesn't get left in a weird state if part of the test fails?

Reply:
True. Let's do this:

```javascript
beforeEach(() => {
  jest.spyOn(console, 'error').mockImplementation(() => {})
})

afterEach(() => {
  console.error.mockRestore()
})
```

Reply:
done